content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+```
+
+```
+root@kali:~/ctf/scans# nmap --script smb-enum-shares,smb-enum-users -p 445 -oA "$ATTACKTARGET_smb_enum" $ATTACKTARGET 
+Starting Nmap 7.80 ( https://nmap.org ) at 2020-11-11 02:52 UTC
+Nmap scan report for ip-10-10-87-38.eu-west-1.compute.internal (10.10.87.38)
+Host is up (0.00077s latency).
+
+PORT    STATE SERVICE
+445/tcp open  microsoft-ds
+MAC Address: 02:FF:5C:EF:7E:F3 (Unknown)
+
+Host script results:
+| smb-enum-shares: 
+|   note: ERROR: Enumerating shares failed, guessing at common ones (NT_STATUS_ACCESS_DENIED)
+|   account_used: <blank>
+|   \\10.10.87.38\ADMIN$: 
+|     warning: Couldn't get details for share: NT_STATUS_ACCESS_DENIED
+|     Anonymous access: <none>
+|   \\10.10.87.38\C$: 
+|     warning: Couldn't get details for share: NT_STATUS_ACCESS_DENIED
+|     Anonymous access: <none>
+|   \\10.10.87.38\IPC$: 
+|     warning: Couldn't get details for share: NT_STATUS_ACCESS_DENIED
+|_    Anonymous access: READ
+
+Nmap done: 1 IP address (1 host up) scanned in 2.80 seconds
+```
+
+```
+root@kali:~/ctf/scans# nmap --script vuln -p 445 -oA "$ATTACKTARGET_smb_vuln" $ATTACKTARGET 
+Starting Nmap 7.80 ( https://nmap.org ) at 2020-11-11 02:53 UTC
+Nmap scan report for ip-10-10-87-38.eu-west-1.compute.internal (10.10.87.38)
+Host is up (0.0011s latency).
+
+PORT    STATE SERVICE
+445/tcp open  microsoft-ds
+|_clamav-exec: ERROR: Script execution failed (use -d to debug)
+MAC Address: 02:FF:5C:EF:7E:F3 (Unknown)
+
+Host script results:
+|_samba-vuln-cve-2012-1182: NT_STATUS_ACCESS_DENIED
+|_smb-vuln-ms10-054: false
+|_smb-vuln-ms10-061: NT_STATUS_ACCESS_DENIED
+| smb-vuln-ms17-010: 
+|   VULNERABLE:
+|   Remote Code Execution vulnerability in Microsoft SMBv1 servers (ms17-010)
+|     State: VULNERABLE
+|     IDs:  CVE:CVE-2017-0143
+|     Risk factor: HIGH
+|       A critical remote code execution vulnerability exists in Microsoft SMBv1
+|        servers (ms17-010).
+|           
+|     Disclosure date: 2017-03-14
+|     References:
+|       https://technet.microsoft.com/en-us/library/security/ms17-010.aspx
+|       https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-0143
+|_      https://blogs.technet.microsoft.com/msrc/2017/05/12/customer-guidance-for-wannacrypt-attacks/
+
+Nmap done: 1 IP address (1 host up) scanned in 15.51 seconds
+```
+
+And once again the scans lead to the same conclusion that the host is vulnerable to eternalblue. So the next step was to hop on google and [exploitdb](https://www.exploit-db.com) to start looking for an exploit. 
+
+
+### Preparing the attack
+The first result that turned up was an exploit from Sleepya [Microsoft Windows 7/2008 R2 - 'EternalBlue' SMB Remote Code Execution (MS17-010) ](https://www.exploit-db.com/exploits/42031), while google pointed me to [worawit/MS17-010](https://github.com/worawit/MS17-010/). On further inspection they looked to be the same exploit however the worawit link provided shellcode that could be used as the basis for a payload.
```

Comment:
whereas google pointed me to...

upon further inspection, they looked like the same exploit, but the worawit link...